pygmt/src/image.py

```diff
@@ -43,11 +43,11 @@ def image(self, imagefile, **kwargs):
         [**g**\|\ **j**\|\ **J**\|\ **n**\|\ **x**]\ *refpoint*\ **+r**\ *dpi*\
         **+w**\ [**-**]\ *width*\ [/*height*]\ [**+j**\ *justify*]\
         [**+n**\ *nx*\ [/*ny*] ]\ [**+o**\ *dx*\ [/*dy*]].
-        Sets reference point on the map for the image.
+        Set reference point on the map for the image.
     box : bool or str
         [**+c**\ *clearances*][**+g**\ *fill*][**+i**\ [[*gap*/]\ *pen*]]\
         [**+p**\ [*pen*]][**+r**\ [*radius*]][**+s**\ [[*dx*/*dy*/][*shade*]]].
-        Without further arguments, draws a rectangular border around the image
+        If set to ``True``, draws a rectangular border around the image
         using :gmt-term:`MAP_FRAME_PEN`.
```

Comment:
Not sure, if we should change "draws" to "draw" here?

Reply:
I think it should be "draw".

Reply:
Thanks! This is changed in commit https://github.com/GenericMappingTools/pygmt/pull/2376/commits/3234d7679ff1777c0c0826dff31e5a4931c6d580.